### ISSUE TITLE
PythonScriptProvider refactored in Ghidra 11.2 

### ISSUE THREAD

**Issue by v-p-b** [dod-cyber-crime-center/pyhidra#43]
Ghidra 11.2 renamed `PythonScriptProvider`, this breaks the `PyScriptProvider` class of pyhidra:

https://scrapco.de/ghidra_docs/javadoc/ghidra/jython/JythonScriptProvider.html

Exception:
```
/path/to/pyhidra/java/plugin/PyScriptProvider.java:25: error: cannot find symbol
public final class PyScriptProvider extends PythonScriptProvider {
                                            ^
  symbol: class PythonScriptProvider
```

**Comment by dc3-tsd** [comment]
Thank you for reaching out about this and we're sorry for the confusion.

This change of behavior was a side effect of our efforts to integrate Pyhidra into Ghidra directly as PyGhidra. Some of the changes that made this work possible were put into the 11.2 release of Ghidra and were backwards breaking for Pyhidra. As such we just uploaded Pyhidra 1.3.0 which includes a set of changes that will allow us to keep it running in the current state until PyGhidra goes fully live as a native part of Ghidra.

Please let us know if this issue is resolved for you in Pyhidra 1.3.0.

**Comment by v-p-b** [comment]
1.3.0 resolves this for me thanks!
